CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1] - (unreleased)
+### Fixed
+- Add missing `composer test` command and use it in CI
+- Fix deprecated usage of prophesize in phpunit tests
+ - Drop support for PHP version 7.2
```

Comment:
```suggestion
- Drop support for PHP version 7.2
```

please move this point up above the `### Fixed` heading - it is not something we fixed, just changed ;-)

Reply:
👍🏻